Core/PSPLoaders.cpp

```diff
@@ -296,6 +296,7 @@ bool Load_PSP_ELF_PBP(FileLoader *fileLoader, std::string *error_string) {
 	// Turn the slashes back to the Windows way.
 	path = ReplaceAll(path, "/", "\\");
 #endif
+	char title[1024];
```

Comment:
This shouldn't be all the way up here if it's not used until much later.

-[Unknown]